src/Microsoft.DocAsCode.Build.Engine/TemplatePreprocessors/TemplateJintPreprocessor.cs

```diff
@@ -86,14 +87,16 @@ private Engine SetupEngine(ResourceCollection resourceCollection, TemplatePrepro
             var rootPath = (RelativePath)scriptResource.ResourceName;
             var engineCache = new Dictionary<string, Engine>();
 
-            var utility = new TemplateUtility(context);
-            object utilityObject = new
+            if (_utilityObject == null)
             {
-                resolveSourceRelativePath = new Func<string, string, string>((s1, s2) => utility.ResolveSourceRelativePath(s1, s2))
-            };
+                var utility = new TemplateUtility(context);
```

Comment:
>context [](start = 50, length = 7)

`_utilityObject` is static but `context` is not. Will context change between different calls of this method?